soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -123,4 +123,41 @@ config IPG_DIV
 	int "IPG clock divider"
 	range 0 3
 
+menuconfig BOOT_HEADER
```

Comment:
Should this be NXP_IMX_RT_BOOT_HEADER or something like that

Reply:
Done